packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -258,10 +259,13 @@ export class AtomicSearchBox {
   }
 
   private async triggerSuggestions() {
-    await Promise.all(
-      this.suggestions.map((suggestion) => suggestion.onInput())
+    // eslint-disable-next-line node/no-unsupported-features/es-builtins
+    const settled = await Promise.allSettled(
```

Comment:
I knew about `Promise.all` but not `Promise.allSettled`, neat! will try to keep in mind from now on.